backend/DataBase/sql_queries/queries.py

```diff
@@ -0,0 +1,7 @@
+GET_TRANSACTIONS = "SELECT BT.id, BT.amount, BT.is_deposit, C.category, C.vendor  FROM Bank_Transaction as BT JOIN Transaction_Category as TC ON BT.id = TC.transaction_id JOIN Category as C ON C.id = TC.category_id;"
+ADD_TRANSACTION_TO_TABLE = """INSERT INTO Bank_Transaction (amount,is_deposit) VALUES ({amount},{is_deposit});"""
```

Comment:
make sure you don't have any case of SQL injection.
best practice is to use the %s and inject it with pymysql commands